Dockerfile

```diff
@@ -1,7 +1,14 @@
 FROM apache/airflow:2.10.4-python3.12
 
 USER root
```

Comment:
Install node and yarn into the Docker container, so that we can use them to build the static site.